### ISSUE TITLE
Dropdowns display focus border after opening

### ISSUE THREAD

**Issue by gbdesign2023** [k-next/kirby#5965]
## Description
The representation of the active language in the drop-down menu is confusing, as the first language is outlined and displayed in bright white. The active language, on the other hand, is displayed in an inconspicuous color. I have also noticed that the first element in a drop-down menu is **always** displayed with a blue border. 

**Expected behavior**  
A neutral representation without preselection would be more logical, as is the case with Kirby 3.

**Screenshots**  

Kirby 4: Active language (EN) in the drop-down menu
<img width="170" alt="language" src="https://github.com/getkirby/kirby/assets/140849227/90a6e64f-a724-48e1-936b-50d475c53eaa">

**Kirby 4:** drop-down menu
<img width="318" alt="drop-down1" src="https://github.com/getkirby/kirby/assets/140849227/06673dde-bc51-4087-9cbf-a58f7b9babb4">

**Kirby 3:** drop-down menu
<img width="318" alt="drop-down2" src="https://github.com/getkirby/kirby/assets/140849227/4260dabb-5590-4ba9-8ca8-715565ce52d6">

## Your setup
Kirby 4 RC1 starterkit

**Your system**
 - MacBook
 - macOS 14.1
 - Browser: Chrome 118

**Comment by lukasbestle** [comment]
I guess this is because the dropdown pulls the keyboard focus when it is opened. When the dropdown was opened with the keyboard this makes sense, but maybe we could avoid that behavior for mouse navigation.

**Comment by distantnative** [comment]
I honestly think it's a bigger accessibility advantage of v4 over v3 that it more consistently shows the current focus.

**Comment by bastianallgeier** [comment]
I agree with Nico. I see this as a huge improvement.

**Comment by nilshoerrmann** [comment]
I think the problematic point is not the focus ring which is a great improvement but that the active state is unclear in comparison: the focus ring is way bolder than the active state. This issue should not be closed in my eyes because this is a real usability issue when editing a site with two languages.

**Comment by distantnative** [comment]
But isn't the way stronger indicator for the current language that the whole button says it?

**Comment by gbdesign2023** [comment]
> I think the problematic point is not the focus ring which is a great improvement but that the active state is unclear in comparison: the focus ring is way bolder than the active state. This issue should not be closed in my eyes because this is a real usability issue when editing a site with two languages.

That was exactly my intention in reporting this anomaly here. Since I don't use keyboard navigation, the visual highlighting was confusing for me at that moment. I could not associate the framing with keyboard navigation.
Accessibility is a useful function for all those who see added value in it. However, an advantage is not generally an advantage for everyone.
As a compromise, the option to enable/disable this feature in Config.php would be desirable.

**Comment by nilshoerrmann** [comment]
> But isn't the way stronger indicator for the current language that the whole button says it?

@distantnative, looking at this screenshot I cannot tell which language is currently selected:

![grafik](https://github.com/getkirby/kirby/assets/25466/7fa7b649-44be-4e10-a74b-881038ac80c2)

Visually, I'd say the active language is German which is also currently focussed. 
From using Kirby I know it's English but that's not what I see.

**Comment by gbdesign2023** [comment]
If accessibility is an important argument, wouldn't the use of colors be an additional argument? There are many people who cannot perceive colors properly. The bright blue has no signal effect here, but the frame does. 

I created a custom-panel.css for the panel:
<img width="338" alt="panel" src="https://github.com/getkirby/kirby/assets/140849227/69128f77-9df2-4e58-bc13-789e7b758e5f">

And I designed the language selection in the frontend like this:
<img width="389" alt="backend" src="https://github.com/getkirby/kirby/assets/140849227/0458d848-f71b-43f9-89a9-be0b1f912eb5">

**Comment by afbora** [comment]
I agree with author and Nils. Especially language dropdown focus behavior very confusing.

**Comment by nilshoerrmann** [comment]
I think a selection indicator that's not color based might help here because the color makes the active item step back visually. Example how Linear and other do this with a checkmark:

![grafik](https://github.com/getkirby/kirby/assets/25466/e61ff88d-65ba-45c5-8d9a-e182f86145c9)

https://linear.app/blog/invisible-details

**Comment by nilshoerrmann** [comment]
Similar example from Height for reference: https://height.app/blog/guide-to-build-context-menus#delay

**Comment by gbdesign2023** [comment]
You can even find a comparable example here on Github. 
In the keyboard navigation, only the background color changes, without additional framing.
<img width="379" alt="github-dropdown" src="https://github.com/getkirby/kirby/assets/140849227/9334d7a4-2fdc-4271-a8ee-5d955e3051f4">

**Comment by distantnative** [comment]
This has been solved in the meantime with a clear checkmark indicator next to the currently active dropdown item
